websocket.go

```diff
@@ -35,6 +38,7 @@ var wsServe = func(cfg *wsConfig, handler WsHandler, errHandler ErrHandler) (don
 			}
 		}()
 		defer close(doneC)
+		keepAlive(c, cfg.timeout)
```

Comment:
it's better to enable keepAlive as an config option, may be we should disable it by default